File: PHP/UberBlack.php
<?php 

require_once("Car.php");

class UberBlack extends Car {
    public $typeCarAccepted; //del hijo
    public $seatsMaterial; //del hijo

    public function __construct(
        $license, //del padre
        $driver, //del padre
        $typeCarAccepted, //del hijo
        $seatsMaterial //del hijo
    ){
        parent::__construct($license, $driver);
        $this->typeCarAccepted = $typeCarAccepted;
        $this->seatsMaterial = $seatsMaterial;
    }
}

File: Java/UberBlack.java
import java.util.ArrayList;
import java.util.Map;

//Clase hija
class UberBlack extends Car {
    Map< String, Map< String, Integer>> typeCarAccepted;
    ArrayList<String> seatsMaterial;

    //Construcctor
    public UberBlack(
            String licence,
            Account driver,
            Map< String, Map< String, Integer>> typeCarAccepted,
            ArrayList<String> seatsMaterial
    ){
        super(licence, driver);
        this.typeCarAccepted = typeCarAccepted;
        this.seatsMaterial= seatsMaterial;
    }
}
